js-with-links.js

```diff
@@ -0,0 +1,78 @@
+// const visit = require('unist-util-visit');
+// var remark = require('remark');
+// var parse = require('remark-parse');
+// var html = require('remark-html');
+// var stringify = require('remark-stringify');
+// var fs = require('fs');
+
+// var file = fs.readFileSync('./src/content/api/node.md', { encoding: 'utf8' });
+
+// remark()
+//   .use(parse)
+//   .use(jsWithLinks)
+//   .use(html)
+//   .process(file, function(err, file) {
+//     if (file) {
+//       console.log('----------------');
+//       console.log(file.contents);
+//       console.log('----------------');
+//     }
+//   });
+
+// module.exports = jsWithLinks;
+
+// function jsWithLinks(options) {
+//   return function transformer(tree, file) {
+//     visit(tree, 'code', visitor);
+
+//     function visitor(node) {
+//       console.log(node)
+//       remark()
+//         .use(function() {
+//           return function t(tr) {
+//             // tr.children.forEach(ch => console.log(ch))
+//           };
+//         })
+//         .use(html)
+//         .process(node.value, (err, file) => {
+//           // console.log('file')
+//           // console.log(file)
+//           // console.log(node.value)
+//           // return file.contents;
+//           node.value = file.toString()
+//         });
+
+//         // console.log(node)
+//     }
+
+//     // console.log('t',transformed);
+//   };
+// }
+
+// function logType(tree) {
+//   console.log(JSON.stringify(tree, null, 2));
+// }
+
+module.exports = jslinks;
+
+jslinks.displayName = 'jslinks';
```

Comment:
It seems this property should be the same as the code snippet language.

Reply:
Should we keep all the commented code?

Reply:
From the original PR text:
> File js-with-links.js will be moved to https://github.com/montogeek/remark-refractor as well.

It means commented code will be deleted, as it is right now: https://github.com/montogeek/remark-refractor/blob/master/src/index.js